src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +45,35 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo?.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Try to write the code so that the linter does not fight and try not to turn off the linter.